src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -135,7 +135,7 @@ function ($a, $b) {
 
         $tpl_data['subtest'] = $request->getAttribute("pageclass")->page ?? null;
 
-        $page = $request->getAttribute("pageclass");
+        $page = $request->getAttribute("pageclass") ?? '';
```

Comment:
this made it so I could view other smarty problems. @driusan maybe I should undo it?